packages/generators/app/lib/resources/templates/database-templates/js/database.template

```diff
@@ -0,0 +1,55 @@
+const path = require('path');
+
+module.exports = ({ env }) => {
+	const client = env('DATABASE_CLIENT', 'sqlite')
+
+	const connections = {
+		mysql: {
+			connection: env('DATABASE_URL') || {
+		    host: env('DATABASE_HOST', 'localhost'),
+		    port: env.int('DATABASE_PORT', 3306),
+		    database: env('DATABASE_NAME', 'strapi'),
+		    user: env('DATABASE_USERNAME', 'strapi'),
+		    password: env('DATABASE_PASSWORD', 'strapi'),
+		    ssl: env.bool('DATABASE_SSL', false) && {
+          key: env('DATABASE_SSL_KEY', undefined),
+          cert: env('DATABASE_SSL_CERT', undefined),
+          ca: env('DATABASE_SSL_CA', undefined),
+          capath: env('DATABASE_SSL_CAPATH', undefined),
+          cipher: env('DATABASE_SSL_CIPHER', undefined),
+          rejectUnauthorized: env.bool('DATABASE_SSL_REJECT_UNAUTHORIZED', true),
+        },
+			},
+		},
+		postgres: {
```

Comment:
missing schema which by default is "public"

Reply:
I can't find it in the Knex connection typings 🤔 It wasn't present in the current implementation of the project generator.

Reply:
Only used for PostgreSQL, in Knex up in the top right make sure you select PG.

Reply:
This isn't set in the current implementation. Is this something that is needed for the database connection to function properly?

Reply:
You can define the schema in the config but it's not part of knex. It is not natively supported but requires calling .withSchema(). 

We support it ourselves in Strapi.

Reply:
I added a schema property on postgres that uses the `DATABASE_SCHEMA` env variable. It is set to `'public'` by default. However the env variable isn't set in the `.env` file when generating a project.

I think that this not being the norm having it available and documented in the documentation should be enough.